dali/operators/sequence/optical_flow/turing_of/optical_flow_turing_test.cc

```diff
@@ -45,24 +46,23 @@ class OpticalFlowTuringKernelTest : public ::testing::Test {
   template<bool Grayscale, typename ColorConversion>
   void ColorConversionTest(ColorConversion cc, std::vector<uint8_t> input_data,
                            std::vector<uint8_t> reference_data) {
-    uint8_t *input, *tested;
-    CUDA_CALL(cudaMallocManaged(&input, input_data.size()));
-    CUDA_CALL(cudaMallocManaged(&tested, reference_data.size()));
-    CUDA_CALL(cudaMemcpy(input, input_data.data(), input_data.size(), cudaMemcpyDefault));
+    DeviceBuffer<uint8_t> input, tested;
+    std::vector<uint8_t> tested_host;
+    input.from_host(input_data);
+    tested.resize(reference_data.size());
+    tested_host.resize(reference_data.size());
 
     auto w = Grayscale ? width_gray_ : width_;
     auto h = Grayscale ? height_gray_ : height_;
     auto p = Grayscale ? pitch_gray_ : pitch_;
 
-    cc(input, tested, p, w, h, 0);
+    cc(input.data(), tested.data(), p, w, h, 0);
```

Comment:
Same here - the output array is not initialized and the pitch is larger than width.

Reply:
Done